stdlib/dataclasses.pyi

```diff
@@ -217,7 +217,13 @@ def is_dataclass(obj: Any) -> bool: ...
 
 class FrozenInstanceError(AttributeError): ...
 
-class InitVar(Generic[_T]):
+if sys.version_info >= (3, 9):
+    _InitVarMeta = type
```

Comment:
```suggestion
    _InitVarMeta: TypeAlias = type
```

Reply:
Sadly, both do not work 😢 
See https://github.com/python/mypy/issues/13334

Reply:
:(

For now, you could probably put the whole class under a `sys.version_info` branch? The `__class_getitem__` method was added in 3.9 as well